configuration/uart.md

```diff
@@ -8,7 +8,7 @@ In Linux device terms, by default, /dev/ttyS0 refers to to the mini-UART, and /d
 
 ## Mini-UART and CPU Core frequency
 
-The baud rate of the Mini-UART is linked to the core frequency of the VPU on the VC4 GPU. This means that as the VPU frequency governor varies the core frequency, the baud rate of the UART also changes. This makes the UART of limited use in the default state. Also, when the linux console uses the mini-UART (Raspberry Pi 3, ZeroW), by default this is disabled for this very reason. 
+The baud rate of the Mini-UART is linked to the core frequency of the VPU on the VC4 GPU. This means that as the VPU frequency governor varies the core frequency, the baud rate of the UART also changes. This makes the UART of limited use in the default state. Also, when the linux console uses the mini-UART (Raspberry Pi 3, ZeroW), this varying baud rate means the console, by default, is disabled. 
```

Comment:
This section goes a bit wrong at the end because the causality gets muddled - yes, the console is disabled by default on 3/0W, but only because the UART is disabled.